Makefile

```diff
@@ -111,6 +112,30 @@ test-sagemaker: build-tests
 	--region ${REGION} \
 	--domain ${AWS_DOMAIN}
 
+
+test-sagemaker-history-server: build-tests
```

Comment:
Adding a separate test for easier and faster iteration of integ tests. The original target `test-sagemaker` remains